docs/docsite/rst/dev_guide/testing_httptester.rst

```diff
@@ -0,0 +1,71 @@
+**********
+httptester
+**********
+
+.. contents:: Topics
+
+Overview
+========
+
+``httptester`` is a docker container used to host certain resources required by :doc:`testing_integration`, this is to avoid CI tests requiring external resources, such as git or package repos, which if temporarily unavailable would cause tests to fail.
+
+HTTP Testing endpoint which provides the following capabilities:
+* httpbin
+* nginx
+* SSL
+* SNI
+
+
+Source files can be found at `test/utils/docker/httptester/ <https://github.com/ansible/ansible/tree/devel/test/utils/docker/httptester>`_
+
+Building
+========
+
+Docker
+------
+
+Both ways of building docker utilize the ``nginx:alpine`` image, but can
+be customized for ``Fedora``, ``Red Hat``, ``CentOS``, ``Ubuntu``,
+``Debian`` and other variants of ``Alpine``
+
+When utilizing ``packer`` or configuring with ``ansible-playbook``
```

Comment:
Commas placement:

 When utilizing ``packer`` or configuring with ``ansible-playbook``, the services will not automatically start on launch and will have to be...
 +manually started using::